### PR TITLE
QALD9: Portuguese string correction

### PR DESCRIPTION
**Pull request by nikit91**
As discussed in https://github.com/ag-sc/QALD/issues/22, the Portuguese question strings are replaced with the strings mentioned in the [spreadsheet](https://docs.google.com/spreadsheets/d/1NMo4bS49eseQt47MH4cA5GoajZySt2gNOzmO1eoOS3I/edit#gid=550865483). 

The replaced string is the last (column-wise) non-empty value present for each row.

### CODE REVIEW

**Comment by RicardoUsbeck** [comment]
@arademaker please check the pull request.

**Comment by RicardoUsbeck** [comment]
Ping @arademaker

**Comment by arademaker** [comment]
I will take a look until begin next week

File: 9/data/qald-9-test-multilingual.json [hunk 1]
```diff
@@ -31990,7 +31990,7 @@
       "keywords" : "Веб-сайт ,   Компания ,   наемный рабочий ,   Больше чем +500000 "
     }, {
       "language" : "pt",
-      "string" : "Dar mim a websites do empresas com Mais do que 500000 funcionários. ",
+      "string" : "Liste para mim os sítios web das companhias com mais de 500000 empregados.",
```
Comment (arademaker) [inline 9/data/qald-9-test-multilingual.json:31993]: This is a European Portuguese translation. In Brazil we would say 

> Liste para mim os sites das companhias com mais de 50 mil funcionários.

File: 9/data/qald-9-test-multilingual.json [hunk 2]
```diff
@@ -31789,7 +31789,7 @@
       "keywords" : "пеший туризм трассы ,   большой каньон ,   нет Опасность из вспышка наводнения "
     }, {
       "language" : "pt",
-      "string" : "exposição mim caminhada trilhas dentro a Grande Canyon Onde há não perigo do instantâneo inundações. ",
+      "string" : "Leve-me às trilhas do Grand Canyon em que não há perigo de inundações.",
```
Comment (arademaker) [inline 9/data/qald-9-test-multilingual.json:31792]: acho que não seria "leve-me" mas "mostre-me"

File: 9/data/qald-9-test-multilingual.json [hunk 3]
```diff
@@ -30856,7 +30856,7 @@
       "keywords" : "мост ,   одна и та же тип ,   Манхеттен Мост "
     }, {
       "language" : "pt",
-      "string" : "Qual pontes estamos do a mesmo tipo Como a Manhattan Ponte? ",
+      "string" : "Quais pontes são do mesmo tipo da ponte de Manhattan?",
```
Comment (arademaker) [inline 9/data/qald-9-test-multilingual.json:30859]: quais pontes são do mesmo tipo que a ponte de Manhattan?

File: 9/data/qald-9-test-multilingual.json [hunk 4]
```diff
@@ -30428,7 +30428,7 @@
       "keywords" : "называется ,   Rodzilla "
     }, {
       "language" : "pt",
-      "string" : "Quem estava chamado Rodzilla? ",
+      "string" : "Quem chamava Rodzilla?",
```
Comment (arademaker) [inline 9/data/qald-9-test-multilingual.json:30431]: quem era chamado Rodzilla?

File: 9/data/qald-9-test-multilingual.json [hunk 5]
```diff
@@ -29886,7 +29886,7 @@
       "keywords" : "Битлз ,   запись ,   первый альбом ,   студия "
     }, {
       "language" : "pt",
-      "string" : "Dentro qual estúdio fez a Beatles registro deles primeiro álbum? ",
+      "string" : "Em qual estúdio os BEatles gravaram o primeiro albúm?",
```
Comment (arademaker) [inline 9/data/qald-9-test-multilingual.json:29889]: Beatles (see capitalization)

File: 9/data/qald-9-test-multilingual.json [hunk 6]
```diff
@@ -29819,7 +29819,7 @@
       "keywords" : " песня из лед а также Огонь "
     }, {
       "language" : "pt",
-      "string" : "Quem é a romancista do a trabalhos uma canção do gelo e fogo? ",
+      "string" : "Qual o romancista que escreveu as Crônicas de gelo e fogo?",
```
Comment (arademaker) [inline 9/data/qald-9-test-multilingual.json:29822]: Crônicas de Gelo e Fogo? (capitalization)

File: 9/data/qald-9-test-multilingual.json [hunk 7]
```diff
@@ -29151,7 +29151,7 @@
       "keywords" : "известный боевой ,   1836 ,   Сан - Антонио "
     }, {
       "language" : "pt",
-      "string" : "o que estava a nome do a famoso batalha dentro 1836 dentro San Antonio? ",
+      "string" : "Qual o nome da famosa batalha em 1863 que ocorre em Santo Antônio?",
```
Comment (arademaker) [inline 9/data/qald-9-test-multilingual.json:29154]: Qual o nome da famosa batalha que ocorre em Santo Antônio em 1863?

File: 9/data/qald-9-test-multilingual.json [hunk 8]
```diff
@@ -28572,7 +28572,7 @@
       "keywords" : "жена ,   президент ,   обама ,   называется ,   Michelle "
     }, {
       "language" : "pt",
-      "string" : "É a esposa do Presidente Obama chamado Michelle? ",
+      "string" : "O nome da esposa do presidente Obama se chamaMichelle?",
```
Comment (arademaker) [inline 9/data/qald-9-test-multilingual.json:28575]: O nome da esposa do presidente Obama é Michelle?

File: 9/data/qald-9-test-multilingual.json [hunk 9]
```diff
@@ -28505,7 +28505,7 @@
       "keywords" : "большой ,   Земля ,   диаметр "
     }, {
       "language" : "pt",
-      "string" : "Como grande é a terra diâmetro? ",
+      "string" : "O quão grande é o diâmetro da Terra?",
```
Comment (arademaker) [inline 9/data/qald-9-test-multilingual.json:28508]: Qual o tamanho do diâmetro da Terra?

File: 9/data/qald-9-test-multilingual.json [hunk 10]
```diff
@@ -20332,7 +20332,7 @@
       "keywords" : "Парагвай ,   провозглашать ,   независимость "
     }, {
       "language" : "pt",
-      "string" : "Quando fez Paraguai proclamar Está independência? ",
+      "string" : "Quando o paraguay proclamou a independência?",
```
Comment (arademaker) [inline 9/data/qald-9-test-multilingual.json:20335]: Paraguay (capitalization)

File: 9/data/qald-9-test-multilingual.json [hunk 11]
```diff
@@ -20260,7 +20260,7 @@
       "keywords" : "школа имя ,   Обама жена ,   изучение "
     }, {
       "language" : "pt",
-      "string" : "o que é a nome do a escola Onde Obama's esposa estudou? ",
+      "string" : "Qual o nome da universidade que a esposa so Obama estudou?",
```
Comment (arademaker) [inline 9/data/qald-9-test-multilingual.json:20263]: Qual o nome da universidade que a esposa **de** Obama estudou?

File: 9/data/qald-9-test-multilingual.json [hunk 12]
```diff
@@ -19913,7 +19913,7 @@
       "keywords" : "дочь ,   Британская граф ,    ,   одна и та же ,   место ,   Родился "
     }, {
       "language" : "pt",
-      "string" : "Qual filhas do britânico condes morreu dentro a mesmo Lugar, colocar eles estavam nascermos dentro? ",
+      "string" : "Qual das filhas da British earls morreu no mesmo local de nascimento?",
```
Comment (arademaker) [inline 9/data/qald-9-test-multilingual.json:19916]: Quais filhas de condes britânicos morreram no mesmo lugar em que nasceram?

**Review by arademaker — CHANGES_REQUESTED** [review]
this is a partical review of some PT translations. My suggestions would be to accept this PR and than I can make a new PR with more reviews, does it work?

**Comment by nikit91** [comment]
@arademaker if you can add your corrections to this [sheet](https://docs.google.com/spreadsheets/d/1NMo4bS49eseQt47MH4cA5GoajZySt2gNOzmO1eoOS3I/edit#gid=550865483) in a new column, then I can quickly update them in this PR. I have scripts written that will update the queries using the latest correction for each of these entries.

**Comment by arademaker** [comment]
I would prefer to work in the text file directly, if possible. I am much more efficient with my emacs than using Google spreadsheet! ;-)

**Comment by arademaker** [comment]
Besides that, having PRs accepted is a way to formally collaborate in a public repository, right? ;-)

**Comment by nikit91** [comment]
well, fair point. I guess then @RicardoUsbeck could accept this PR and afterwards you (@arademaker) can raise a separate PR with corrections :)

**Comment by RicardoUsbeck** [comment]
I am actually not sure what is happening here. If I just click 'resolve' on all comments from @arademaker shouldn't that be it? Why would he need to raise another PR, @nikit91 ?

**Comment by arademaker** [comment]
I made comments, no updates on the files. Someone need to apply my comments changing this PR (new commit) and accepting it. 

I do see more corrections to make. So I will open a new PR to that after this PR is accepted. 

Alternatively, ignore my comments. Accept this PR and I later make a new PR to fix directly the files with the erros I already mentioned in the comments and new ones that I know to exist.

**Comment by RicardoUsbeck** [comment]
Thanks for the super fast reply. I will accept this PR as is and leave it to your emacs magic to create a new PR.

**Pull request merged**